### ISSUE TITLE
population$setParameterValues fails for populations of size 1

### ISSUE THREAD

**Issue by Siel** [Open-Systems-Pharmacology/OSPSuite-R#427]
The simulation stills runs with population sizes of 1 so I don't think it's the intended behaviour. Even if that's the case the error message is not clear.

My current workaround is to set the parameter value to the simulation object:

`setParameterValuesByPath(path, value, simulation = sim)`

**Comment by msevestre** [comment]
@Siel A population of one does not really make sense 
see 
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/424#issuecomment-665634991

We'll fix this issue in the next release but if you only have one individual, use the setParameterValuesByPAth indeed. You won't have the overhead related to population simulation

**Comment by Siel** [comment]
Yes, I understand that. In our case is it possible to have populations of only one individual. But you're right, they would be easily detected so we can switch to setParameterValuesByPath.

Thanks

**Comment by IndrajeetPatil** [comment]
@Siel Can this issue be closed?

**Comment by Siel** [comment]
Yes, you can close it, thanks

**Comment by msevestre** [comment]
Hum I am not sure if this should be closed. The error message should be clearer
Or has this been fixed?

**Comment by msevestre** [comment]
see comment here
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/427#issuecomment-692987993
